### ISSUE TITLE
[BUG] Defragmentation failed due to certificate for 0.0.0.0 unable to validate

### ISSUE THREAD

**Issue by ishan16696** [gardener/etcd-druid#325]
**Describe the bug:**
The [Client.Status](https://github.com/gardener/etcd-backup-restore/blob/3fa9156cc70ec205aa7a62f21d6ea33b7c6c4114/pkg/miscellaneous/miscellaneous.go#L217) API calls fails in [defragmentor](https://github.com/gardener/etcd-backup-restore/blob/3fa9156cc70ec205aa7a62f21d6ea33b7c6c4114/pkg/defragmentor/defrag.go#L74-L84) due to `authentication handshake failed: x509: cannot validate certificate for 0.0.0.0 because it doesn't contain any IP SANs`

**Expected behaviour:**
The Client.Status API call Shouldn’t fail.

**Logs:**
```
{"level":"warn","ts":"2022-04-05T04:31:01.641Z","caller":"clientv3/retry_interceptor.go:62","msg":"retrying of unary invoker failed","target":"passthrough:///https://0.0.0.0:2379/","attempt":0,"error":"rpc error: code = DeadlineExceeded desc = latest balancer error: connection error: desc = \"transport: authentication handshake failed: x509: cannot validate certificate for 0.0.0.0 because it doesn't contain any IP SANs\""}
time="2022-04-05T04:31:01Z" level=error msg="failed to get status of etcd endPoint: https://0.0.0.0:2379/ with error: context deadline exceeded" actor=backup-restore-server job=defragmentor
time="2022-04-05T04:31:01Z" level=error msg="failed to defrag as all members of etcd cluster are not healthy: context deadline exceeded" actor=backup-restore-server job=defragmentor
time="2022-04-05T04:31:01Z" level=info msg="All etcd members endPoints: [https://0.0.0.0:2379/]" actor=backup-restore-server job=defragmentor
```

**Environment (please complete the following information):**
 - Etcd version/commit ID :
 - Etcd-druid version/commit ID : `v.0.8.1`
 - Cloud Provider [All/AWS/GCS/ABS/Swift/OSS]: ALL

**Anything else we need to know?**:

**Comment by ishan16696** [comment]
I think we are getting the endpoints to apply the defragmentation from [MemberList](https://github.com/gardener/etcd-backup-restore/blob/3fa9156cc70ec205aa7a62f21d6ea33b7c6c4114/pkg/miscellaneous/miscellaneous.go#L196) API call which gets the endpoints of etcd from [Etcd-configuration](https://github.com/gardener/etcd-druid/blob/hotfix-v0.8/charts/etcd/templates/etcd-configmap.yaml#L44-L48) which was passed via configMap.

**Comment by ishan16696** [comment]
For Master branch this issue of `0.0.0.0` or `127.0.0.1` not present in certificate `SANs list` has been fixed by this [PR#314](https://github.com/gardener/etcd-druid/pull/314)

**Comment by ishan16696** [comment]
It is fixed by this PR https://github.com/gardener/etcd-backup-restore/pull/459 for backup-restore`v0.15.x`.
And for master branch https://github.com/gardener/etcd-druid/pull/314
